main/lregex.c

```diff
@@ -2753,7 +2755,7 @@ static void   guestRequestSubmit (struct guestRequest *r)
 {
 	const char *langName = getLanguageName (r->lang);
 	verbose ("guestRequestSubmit: %s; "
-			 "range: %ld - %ld\n",
+			 "range: %"PRIdPTR" - %"PRIdPTR"\n",
```

Comment:
Hmm, this still causes warnings on 32-bit MinGW.
https://ci.appveyor.com/project/universalctags/ctags/builds/27963569/job/nbs73twybwv5s63l?fullLog=true#L676
It seems that `off_t` is 64-bit on MinGW and 32-bit on Visual C++ regardless of the bitness of the compiler.
Need a something different way...

Reply:
Always casting `off_t` to `long long` might be the easiest way. Then we can use `%lld` here.